front/src/components/device/form/DeviceRoomSelector.jsx

```diff
@@ -0,0 +1,52 @@
+import { Component } from 'preact';
+import { Text } from 'preact-i18n';
+import { connect } from 'unistore/preact';
+import cx from 'classnames';
+
+import actions from '../../../actions/house';
+import { RequestStatus } from '../../../utils/consts';
+
+@connect('houses,housesGetStatus', actions)
```

Comment:
I think there is an issue with the variable name `houses`

As the state is global, it's shared with all other componenents

With have some components that use the same key (`houses`) to store a list of houses without the rooms

It caused some issues in the past, that's why I renamed this key `housesWithRooms` when the room was included

Two ways of fixing this: 

- Rename the key to `housesWithRooms` here
- Stop using global variables (but we are losing the cache)

Reply:
Example here : https://github.com/GladysAssistant/Gladys/blob/master/front/src/components/boxs/weather/EditWeatherBox.jsx#L18